src/test/java/com/bakdata/common_kafka_streams/test_applications/WordCount.java

```diff
@@ -48,7 +48,7 @@ public void buildTopology(final StreamsBuilder builder) {
         final Serde<String> stringSerde = Serdes.String();
         final Serde<Long> longSerde = Serdes.Long();
 
-        final KStream<String, String> textLines = builder.stream(this.inputTopic);
+        final KStream<String, String> textLines = builder.stream(this.inputTopics);
```

Comment:
While this may be correct in this specific use case, it is an unintended way of using the Kafka API. When passing a Collection to `builder.stream`, all input topics are simply merged into a single stream without ordering guarantees. Please use `this.inputTopics.get(0)` to make it more explicit here.